unitTests/ui/Combobox/ComboboxValuesTest.ts

```diff
@@ -8,7 +8,7 @@ function createValuesFromResponse(response: string[]) {
   return response.map(value => {
     return {
       value,
-      element: $$('li', {}, value).el
+      element: $$('div', {}, $$('li', { class: 'coveo-checkbox-span-label' }, value).el).el
```

Comment:
During the test, seems you have found something interesting... it was indeed failing when the parent element had the class and not a child. I think that use case should be supported. You can use a ternary
```
let result = $$(searchResult).hasClass(this.combobox.options.highlightValueClassName)
  ? searchResult 
  : $$(searchResult).find(`.${this.combobox.options.highlightValueClassName}`);
```

You can have 2 tests, one with the class on the parent, one on a child

Reply:
I added a second test with a local function to replace this one. This new function adds a wrapper to have the class on a child node.